.github/workflows/scripts/create-release-packages.sh

```diff
@@ -54,20 +58,43 @@ generate_commands() {
       script_command="(Missing script command for $script_variant)"
     fi
     
-    # Replace {SCRIPT} placeholder with the script command
-    body=$(printf '%s\n' "$file_content" | sed "s|{SCRIPT}|${script_command}|g")
-    
-    # Remove the scripts: section from frontmatter while preserving YAML structure
-    body=$(printf '%s\n' "$body" | awk '
-      /^---$/ { print; if (++dash_count == 1) in_frontmatter=1; else in_frontmatter=0; next }
-      in_frontmatter && /^scripts:$/ { skip_scripts=1; next }
-      in_frontmatter && /^[a-zA-Z].*:/ && skip_scripts { skip_scripts=0 }
-      in_frontmatter && skip_scripts && /^[[:space:]]/ { next }
-      { print }
-    ')
-    
-    # Apply other substitutions
-    body=$(printf '%s\n' "$body" | sed "s/{ARGS}/$arg_format/g" | sed "s/__AGENT__/$agent/g" | rewrite_paths)
+    # Special handling for Kilo Code YAML format
```

Comment:
Is Kilo Code the only one using YAML?

Reply:
Nope...Roo-Code and Cline use YAML as well
So once this works ....the same can be extended for the others as well